src/Domain/Creators/TimeDependence/UniformTranslation.hpp

```diff
@@ -101,6 +112,11 @@ class UniformTranslation final : public TimeDependence<MeshDim> {
   UniformTranslation(double initial_time,
                      const std::array<double, MeshDim>& velocity);
 
+  UniformTranslation(
```

Comment:
Please add some docs to the class explaining what's going on with the two velocities and the distorted frame